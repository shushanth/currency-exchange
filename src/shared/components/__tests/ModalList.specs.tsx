import React from 'react';
import renderer from 'react-test-renderer';

import { CEModalList } from '../index';

describe('<CEModalList />', () => {
  test('tree snapshots match', () => {
    const modalTree = renderer
      .create(
        <CEModalList
          name="test-modal"
          showModal={true}
          onClose={() => {}}
          onItemSelect={() => {}}
        />,
      )
      .toJSON();
    expect(modalTree).toMatchSnapshot();
  });
});
