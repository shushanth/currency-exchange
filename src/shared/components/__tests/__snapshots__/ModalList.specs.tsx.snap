// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CEModalList /> tree snapshots match 1`] = `
<div
  className="sc-fzplWN eTEdGz"
  id="test-modal"
>
  <a
    className="sc-fznZeY dKDDpj"
    onClick={[Function]}
  >
    close
  </a>
  <div
    className="sc-fznyAO hwqMjk"
  />
</div>
`;
